content/en/docs/tasks/administer-cluster/out-of-resource.md

```diff
@@ -18,31 +18,21 @@ nodes become unstable.
 
 {{% /capture %}}
 
-
 {{% capture body %}}
 
-## Eviction Policy
```

Comment:
This is specific to kubelet, so it may make sense to keep it here especially the signals are still here.

Reply:
It depends on how you see this page: is it explaining a task and hyperlinking to other pages to provide context? Or is this page explaining how a thing works?

If the page is explaining how a thing works, it usually belongs in Reference or Concepts rather than in Tasks.